uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/AbsencePeriodTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_Test_Fabricator_AbsencePeriod as AbsencePeriodFabricator;
+
+/**
+ * Class api_v3_AbsencePeriodTest
+ *
+ * @group headless
+ */
+class api_v3_AbsencePeriodTest extends BaseHeadlessTest {
+
+  public function testValidPeriodExistsReturnsFalseWhenThereIsNoAbsencePeriodWithEndDateGreaterThanOrEqualToCurrentDate() {
+    AbsencePeriodFabricator::fabricate([
+      'start_date' => CRM_Utils_Date::processDate('-5 days'),
+      'end_date'   => CRM_Utils_Date::processDate('yesterday'),
+      'weight'    => 1
```

Comment:
Is `weight` required in all these calls, or what does it do?

Reply:
Will yank off these tests since we can use the API to retrieve the info after all.